source/core/slang-nvrtc-compiler.cpp

```diff
@@ -309,7 +309,12 @@ SlangResult NVRTCDownstreamCompiler::compile(const CompileOptions& options, RefP
         cmdLine.addArg("-w");
 
         //
+#if 0
         cmdLine.addArg("-arch=compute_70");
+#else
+        // Needed for Warp intrinsics
+        cmdLine.addArg("-arch=compute_30");
+#endif
```

Comment:
If having the `#ifdef`ed out code here intentional?

Reply:
Just so I remember it's a place holder, and that I will have to pass something down to nvrtc depending on what arch is required, based on compiled code.